### PR TITLE
elixir/word-count: Update mentoring.md

### PR DESCRIPTION
**Pull request by Eiermann**
Added a short solution using Enum.frequencies.

### CODE REVIEW

File: tracks/elixir/exercises/word-count/mentoring.md [hunk 1]
```diff
@@ -56,7 +56,18 @@ defmodule WordCount do
...
+defmodule WordCount do
 
+  def count(sentence) do
+    Regex.scan(~R"[[:alnum:]-]+"u, String.downcase(sentence))
+    |> List.flatten
+    |> Enum.frequencies
+  end
+  
+end
+```
```
Comment (SleeplessByte) [inline tracks/elixir/exercises/word-count/mentoring.md:70]:
````suggestion
```

````

File: tracks/elixir/exercises/word-count/mentoring.md [hunk 2]
```diff
@@ -56,7 +56,18 @@ defmodule WordCount do
...
+```elixir
+# Utilizing Enum.frequencies
+defmodule WordCount do
 
+  def count(sentence) do
+    Regex.scan(~R"[[:alnum:]-]+"u, String.downcase(sentence))
+    |> List.flatten
+    |> Enum.frequencies
+  end
+
```
Comment (NobbZ) [inline tracks/elixir/exercises/word-count/mentoring.md:68]:
```suggestion

```

File: tracks/elixir/exercises/word-count/mentoring.md [hunk 3]
```diff
@@ -56,7 +56,18 @@ defmodule WordCount do
   end
 end
 ```
+```elixir
```
Comment (NobbZ) [inline tracks/elixir/exercises/word-count/mentoring.md:59]:
```suggestion

```elixir
```

**Review by NobbZ — CHANGES_REQUESTED** [review]
Even though `Enum.frequencies/1,2` exists, I'm not really sure if it is within "first principles" or a valid use of standard library…

Also we have to remember, that not everyone already has access to elixir 1.10, and therefore they might not have access to this function, as it has been added with 1.10.

Therefore I think it should not only the example added, but the use of `Enum.frequencies/1,2` could be something to talk about, but nothing that would actually block a students progress.

**Review by NobbZ — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Thanks @NobbZ for re-checking.

**Pull request merged**